pybamm/parameters/standard_parameters_lead_acid.py

```diff
@@ -191,3 +210,16 @@
 epsn_init = epsn_max - epsDelta_n * (1 - q_init)  # Initial pororsity (neg) [-]
 epss_init = epss_max  # Initial pororsity (sep) [-]
 epsp_init = epsp_max - epsDelta_p * (1 - q_init)  # Initial pororsity (pos) [-]
+
+
+# Concatenated symbols
+# artificially add domains - doesn't seem like the best way of doing this
+# TODO: look into how to do this more elegantly
+neg = pybamm.Scalar(1, domain=["negative electrode"])
```

Comment:
could just add a piecewise constant function that takes in a list of values and a list of domains and outputs the concatenated scalar object